repo.pl

```diff
@@ -452,6 +452,12 @@ ()
                      $CFG{_PACKAGE_FNAME} = basename( $f[0] );
 
                      $CFG{_PACKAGE_OS} = undef;
+		     $CFG{_PACKAGE_OS} = "UBUNTU18"
```

Comment:
nit - indentation looks slightly off here.  Perhaps mixing tabs and spaces???

Reply:
Yes it was mix of tab and space.